docs/conf.py

```diff
@@ -53,7 +53,7 @@
 # ones.
 extensions = [
     'sphinx.ext.autodoc',
-    'sphinx.ext.autosectionlabel',
```

Comment:
Why is this extension disabled? I think we want to make sure that every section has a label?

Reply:
Because it introduces warnings on duplicate labels. I have added suppressor instead.